scripts/views/eventView.js

```diff
@@ -9,10 +9,30 @@
   eventHandler.lookup = function (e) {
     e.preventDefault();
     var zip = $('#look-up input').val();
-    if (zip) {
-      TownHall.lookupZip($('#look-up input').val());
-      eventHandler.resetFilters();
+    if (zip && zip.length === 5) {
```

Comment:
yeah, I thought about this, but the zip call also has error handling... if the query isn't in our database it returns en error... but I guess there is no reason to not handle it here.